### ISSUE TITLE
you can enter more then 8 decimels in dialogs

### ISSUE THREAD

**Issue by shefass** [btdex/btdex#89]
btc have 8 decimals, burst have 8 decimals, our tokens have max 8 decimals. 

Better restrict entering more then 8 decimals in UI, it will be possibly better UX

**Comment by jjos2372** [comment]
If the user enters numbers with more than 8 decimal places they are converted to 8 decimal places. I don't see a need to change this behavior.

Please comment further or close the issue.

**Comment by benthetechguy** [comment]
I don't think it's about the functionality, but about the UX. This is just to stop people from potentially being confused when they try to enter a 9-decimal-place amount and get denied when clicking ok. If we stop them from entering the 9-decimal-place number in the first place, they wouldn't get confused.

**Comment by goodplanets** [comment]
the qotient of two 8 digit numbers can create a 9 or 10 digit number
the rate for a unit price is not a cryptocurrency regulated number it is derived from the price of two numbers that may have 8 digits there's no rule 8 digits x 8 digits don't = 9digit numbers like 0.00000001 unit price. 

for example if i have 1 satoshi and i wish to ssell it for 10 burst my unit price is 0.000000001

**Comment by goodplanets** [comment]
the unit rate must be able to accomodate for any priced asset from 1 satoshi.

**Comment by goodplanets** [comment]
currently the unit rate can only compute if the numbers in the unit rate field have 8 decimals where as the unit rate of a non fungible token like an satoshi for example could extend to 9 digits.

**Comment by benthetechguy** [comment]
It doesn't matter if your offer requires a price of 0.000000001, that's physically impossible as all the currencies we use have a limit of 8 places, it's literally impossible to buy 0.000000001 BTC or even BURST no matter how hard you try.
Besides, this issue isn't about whether or not we should have more than 8 places, it's about fixing a UI issue where you can type in more than 8 places, but not proceed with the order any further.

**Comment by goodplanets** [comment]
yes ben but it's not impossible to have a unit rate that is greater than 8 decimals despite being limited to an minimum amt

**Comment by goodplanets** [comment]
according to jjos the 9th digit in the unit rate can't be computed or conveyed currently due to the limitations that burst was coded with.

**Comment by benthetechguy** [comment]
I'm not saying you're wrong, i'm saying that's not what this issue is about. If you want to argue your point, open a different issue.

**Comment by goodplanets** [comment]
![image](https://user-images.githubusercontent.com/40472463/88462600-2e80ad80-ce72-11ea-8803-17d3f05a0ce2.png)
